share/man/man4/iwlwififw.4

```diff
@@ -46,10 +46,10 @@ driver.
 .Bl -column -compact "Vendor" "Device" "Subv. " "Subd. " "Firmware-Prefix"
 .It Ar Name
 .It Ar Vendor Ta Ar Device Ta Ar Subv. Ta Ar Subd. Ta Ar Firmware-Prefix
-.% ---------------------------------------------------------------------
-.% This list is manually generated from a sysctl and post-processing.
-.% Edits will be overwritten on next update.
-.% ---------------------------------------------------------------------
+.\" ---------------------------------------------------------------------
```

Comment:
The fixed "begin comment" is exactly how it's done in `share/man/man4/iwlwifi.4`.  That file was fixed in b95fa4203d698063b02b03647cd00906cd94687a, but evidently they didn't notice that the same mistake was in this file.